deps.sh

```diff
@@ -115,7 +115,7 @@ function install_cramfstools
 
 function install_ubireader
 {
-    git clone --quiet --depth 1 --branch "master" https://github.com/jrspruitt/ubi_reader
+    git clone --quiet --depth 1 -b v0.8.5-master https://github.com/jrspruitt/ubi_reader
```

Comment:
Similarly, we could achieve the same effect (without pinning to an outdated tag) by just switching `main` to `master`. Either way, I support this change to un-break the `deps.sh` script!

```suggestion
    git clone --quiet --depth 1 --branch "main" https://github.com/jrspruitt/ubi_reader
```

Reply:
This isn't enough anymore, since newer version of ubi_reader use Poetry instead of setuptools (there's no setup.py).

I think the correct solution here is to edit `deps.sh` to install ubi_reader from PyPi instead of source (i.e. run `pip install ubi_reader`).